app/google-services.json

```diff
@@ -0,0 +1,40 @@
+{
+  "project_info": {
+    "project_number": "828198983391",
+    "firebase_url": "https://projmag-default-rtdb.europe-west1.firebasedatabase.app",
+    "project_id": "projmag",
+    "storage_bucket": "projmag.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:828198983391:android:ea2db168dfb5014a5dbb6a",
+        "android_client_info": {
+          "package_name": "com.sdp13epfl2021.projmag"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "828198983391-lln2rum7cfhhe43uv68laa9rkntoe62c.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyAc17HYqLJVpYHlVSNEIZ4ObmN40Pl5H2U"
```

Comment:
Wouldn't it be safer to put the API key in a github secret (like in SwEng bootcamp) ?